docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -200,28 +200,70 @@ Once the file storage is added and enabled for projects, your users are able to
+### Setting up OpenProject in Nextcloud
 
-- If you are on a local network with your Nextcloud and OpenProject hosts, it might be necessary to explicitly allow local remote servers by setting a system configuration flag on your Nextcloud host's command line. Use the command `sudo -u www-data php occ config:system:set allow_local_remote_servers --value 1` in order to change the setting for Nextcloud.
+#### Error message "Please enter a valid OpenProject hostname"
 
-- To test the connection between OpenProject and Nextcloud, you can use the following methods from your OpenProject host's command line:
+On Nextcloud inside the _OpenProject Integration_ App, when adding the OpenProject host, it shows the error **"Please enter a valid OpenProject hostname"**. There are multiple reasons that can cause the error;
+
+- The host address could be wrong. Please check in a separate browser tab that the URL that you have provided is correct. OpenProject should load. 
+- If your Nextcloud and OpenProject servers are both in a local network, it might be necessary to explicitly [allow local remote servers in Nextcloud](https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html) by setting a system configuration via your Nextcloud command line. This allows Nextcloud to connect to other local servers. Use the command `sudo -u www-data php occ config:system:set allow_local_remote_servers --value 1` in order to enable the setting for Nextcloud.
+- Your Nextcloud server does not have access to the OpenProject server. To test the connection between Nextcloud and OpenProject, you can use the following command on your Nextcloud server's command line:
 
   ```bash
-  curl --location --request GET 'https://nextcloud.example.com/index.php/apps/integration_openproject/check-config' --header 'Authorization: foo'
-  
-  # If Nextcloud OpenProject Integration is integrated correctly the response should look like the following
-  
-  {"user_id":"","authorization_header":foo}
+  curl https://openproject.example.com
   ```
-  
+  In case the `curl` command above results in an error pay attention what it is telling you. Typical reasons for connection issues are misconfigured fire walls, proxies, or a bad TLS/SSL setup.
+
+##### While using a self signed TLS/SSL certificate you receive "certificate verify failed"
+
+Some administrators setup OpenProject using a self signed TLS/SSL certificate with their own CA (certificate authority). The CA needs to be known on the Nextcloud server. On a Debian/Ubuntu based server, make sure you add the CA certificate for your OpenProject certificate to `/usr/local/share/ca-certificates` and run `sudo update-ca-certificates` afterwards. Then Nextcloud's PHP code should be able to verify your Nextcloud TLS/SSL certificate when emitting HTTPS requests to your Nextcloud server.
```

Comment:
`...emitting HTTPS requests to your Nextcloud server.` -> `...emitting HTTPS requests to your OpenProject server.`

Reply:
Great catch! There is even another place in the sentence where "Nextcloud" needed to be replaced with "OpenProject".